src/main/kotlin/net/axay/blaubot/commands/Admin.kt

```diff
@@ -25,6 +32,16 @@ object Admin : SlashCommand(
         interaction.command.subCommands["chat"]?.options?.get("message")?.string()?.let {
             interaction.channel.createMessage(it)
         }
+        
+        interaction.command.subCommands["embed"]?.options?.get("message")?.string()?.let {
+            interaction.acknowledge(false).followUp {
+                embed {
+                    title = interaction.command.subCommands["embed"]?.options?.get("title")?.string()
+                    image = interaction.member.asUser().avatar.url
```

Comment:
Why do you want to display the avatar of the user who created the embed?

Reply:
I thought you can see who sent the message. I can also replace it with the server icon